tools/export_program_info/export_program_info.vcxproj

```diff
@@ -255,5 +255,14 @@ $(OutputPath)export_program_info.exe &gt; $(OutputPath)export_program_info.log
       <Project>{aa933b9f-b5d8-4aa8-ac18-98fe1a161e8a}</Project>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <ResourceCompile Include="..\..\resource\ebpf_resource.rc">
+      <AdditionalIncludeDirectories Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">$(SolutionDir)include</AdditionalIncludeDirectories>
```

Comment:
Same here that it'd be nice to repeat this less often - either in `Directory.Build.props` or just once without a condition for each config/platform